docs/specification/current.md

```diff
@@ -929,6 +918,21 @@ As another example, an extension defined on the discovery response could look li
 ```
 [OAuth 2.0]: https://oauth.net/2/
 
+## Data Types
+
+CDS Hooks leverages json data types throughout.  This section defines data structures re-used across the specification. 
+
+### Coding
+
+The **Coding** data type captures the concept of a code. A code is understood only when the given code, code-system, and a optionally a human readable display are available. This coding type is a standalone data type in CDS Hooks modeled after a trimmed down version of the [FHIR Coding data type](http://hl7.org/fhir/datatypes.html#Coding).
+
+Field | Optionality | Type | Description
+----- | ----- | ----- | --------
+`code` | REQUIRED | *string* | The code for what is being represented
+`system` | OPTIONAL | *string* | A codesystem for this `code`.
+`display` | OPTIONAL | *string* | A short, human-readable label to display.
+`version` | OPTIONAL | *string* | Version of the system - if relevant.
+`userSelected` | OPTIONAL | *boolean* | If this coding was chosen directly by the user.
```

Comment:
Topics or OverrideReason options returned from the CDS service will never be marked as userSelected.

An override reason sent to the feedback endpoint so far has been echoing back one of the options returned by the CDS service rather than some translation.  Isn’t `userSelected` typically for something like a CodeableConcept, to distinguish whether a specific Coding was directly chosen, to help when translating among multiple synonymous codes from different systems?

Reply:
`userSelected` removed. @dennispatterson - do you want to re-review? We need to merge this today.